src/plugins/saved_objects_finder/public/index.ts

```diff
@@ -8,6 +8,6 @@
 
 import { SavedObjectsFinderPublicPlugin } from './plugin';
 export type { SavedObjectMetaData, SavedObjectFinderProps } from './finder';
-export { SavedObjectFinder } from './finder';
+export { SavedObjectFinder, SavedObjectFinderUi, getSavedObjectFinder } from './finder';
```

Comment:
```suggestion
export { SavedObjectFinder, getSavedObjectFinder } from './finder';
```
Similar to my other comment about lazy loading, but are we able to just use the lazy loaded component instead of including this export which adds ~4kb to the page load bundle?